content/docs/guides/components/istio.md

```diff
@@ -60,16 +60,18 @@ so this service should not be of type `LoadBalancer`.
 - Edit `install/kubernetes/istio-demo.yaml` and change the service type of `istio-ingressgateway` from
   `LoadBalancer` to `NodePort`.
 
-### 4. Install istio (without mTLS)
+### 5. Install istio (without mTLS)
     ```
     kubectl apply -f install/kubernetes/istio-demo.yaml
```

Comment:
This is a pretty brittle and confusing customer experience. I followed these instructions myself and it's nontrivial to install istio, especially for inexperienced users. It also relies on a brittle and strange filename (demo?) and if istio changes anything, these docs will become incorrect.

Instead, I think it would be much more robust to snapshot istio's install. Knative has taken this approach and bundled their default istio install into a file you can apply straight from the web: https://raw.githubusercontent.com/knative/serving/v0.2.1/third_party/istio-1.0.2/istio.yaml

Reply:
Looking at https://github.com/kubeflow/kubeflow/pull/2020/files, It looks like you've done exactly this. Great!

Reply:
Yeah, that's the plan